lib/avro-patches/logical_types/schema.rb

```diff
@@ -57,8 +57,8 @@ def self.real_parse(json_obj, names=nil, default_namespace=nil)
   end
 
   # Determine if a ruby datum is an instance of a schema
-  def self.validate(expected_schema, logical_datum, recursive = true, encoded = false)
-    Avro::SchemaValidator.validate!(expected_schema, logical_datum, recursive, encoded)
+  def self.validate(expected_schema, logical_datum, options = { recursive: true, encoded: false })
```

Comment:
This doesn't work for defaulting if clients pass individual options e.g. `validate(expected_schema, logical_datum, recursive: true)`